index.html

```diff
@@ -533,6 +556,11 @@ <h2>Filter <var>buffer</var> by <var>name</var> and <var>type</var></h2>
         </li>
       </ol>
     </section>
+    <section data-link-for="PerformanceObserver">
+      <h2>Register performance entry type</h2>
+      <p>Given a string <var>type</var>, to register <var>type</var> the user
+        agent MUST append <var>type</var> to <a>supported entry types</a>.</p>
```

Comment:
Can we define a simple algorithm here, instead of these "MUST" semantics?
e.g. "1. Let `type` be the input string to the algorithm 2. Append `type` to `supported entry types`"

Reply:
I don't see the benefit here, it would be a one-step (two if you count mentioning the input as one step) algorithm with no return value... And also this is supposed to be implicitly done by the user agent when implementing a specification, not a normal algorithm that can be called at any time.

Reply:
It's cleaner. Right now IIUC you want to have other specifications link to the section itself. It's cleaner to have them link to an algorithm definition, and for that algorithm to actually be well-defined.
I agree that it is super simple and doesn't return a value, but it'd still be cleaner.


> And also this is supposed to be implicitly done by the user agent when implementing a specification, not a normal algorithm that can be called at any time.

It is supposed to be called from different specifications that implement PO.

Reply:
Ok I wrote it as a 2-step algorithm.